app/views/users/index.erb

```diff
@@ -1,3 +1,34 @@
-<h1 class="govuk-heading-xl"><%= t 'users.title' %></h1>
-
-<%= link_to t('users.invite_button'), invite_user_path , class: "govuk-button", data: { module: "govuk-button" }, role:"button", draggable:"false" %>
+<% if @gds %>
+  <h1 class="govuk-heading-xl"><%= t 'users.title_no_team' %></h1>
+  <div class="govuk-accordion" data-module="govuk-accordion" id="accordion-default">
+    <% @teams.reverse.each do |team| %>
+      <div class="govuk-accordion__section ">
+        <div class="govuk-accordion__section-header">
+          <h2 class="govuk-accordion__section-heading">
+            <span class="govuk-accordion__section-button" id="accordion-default-heading-1">
+              <%= team.name %>
+            </span>
+          </h2>
+        </div>
+        <div id="accordion-default-content-1" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-1">
+          <p class='govuk-body'><%= link_to "#{t('users.invite.button_to')} #{team.name}", invite_to_team_path(team.id), class: 'govuk-button'%></p>
+        </div>
+      </div>
+    <%end%>
+    <div class="govuk-accordion__section ">
+        <div class="govuk-accordion__section-header">
+          <h2 class="govuk-accordion__section-heading">
+            <span class="govuk-accordion__section-button" id="accordion-default-heading-1">
+              Government Digital Service
+            </span>
+          </h2>
+        </div>
+        <div id="accordion-default-content-1" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-1">
+          <p class='govuk-body'><%= link_to "#{t('users.invite.button_to')} GDS", invite_to_team_path(0), class: 'govuk-button'%></p>
```

Comment:
I know it's unlikely we'll have non roman alphabet users, but if we're using a localisation and then also interpolating english letters/words it feels like we should just localise the whole thing.

Reply:
Created a GDS translation under common.